src/main/java/com/bobocode/dto/UrlInfoRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.bobocode.dto;
+
+import javax.validation.constraints.NotBlank;
+
```

Comment:
@kebann what do you think about the following formatting?
```java
public record UrlInfoRequest(
        @NotBlank(message = "cannot be null or blank")
        String url,
        String title
) {
}
```

Reply:
I'll like it. I just went with the default IntelliJ formatting.